src/Microsoft.DotNet.CodeFormatter.Analyzers/OptimizeNamespaceImportsOptions.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Composition;
+
+using Microsoft.CodeAnalysis.Options;
+ 
+namespace Microsoft.DotNet.CodeFormatter.Analyzers
+{
+    [Export(typeof(IOptionsProvider)), Shared]
+    internal class OptimizeNamespaceImportsOptions : IOptionsProvider
```

Comment:
Another note is that while this MEF export mechanism is good for codeformatter, when we get to designing this for analyzers for real, we'd likely need the declaration of options and their defaults to be in DiagnosticDesciptor. For eg: we might add a descriptor.Options which could return an ImmutableArray<PerLanguageOption>